README.md

```diff
@@ -36,15 +36,17 @@ The user can submit transactions to a node which includes them in its local memp
 Write the following commands in a terminal:
 
 1.) enter into a `nix-shell`
+
```

Comment:
Maybe here add an instruction to build the `cardano-node` executable with `stack` or put the command in `demo.sh`?